webrtc.html

```diff
@@ -4132,30 +4132,23 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
             the <a><code>RTCRtpReceiver</code></a> to be stopped.</p>
           </dd>
 
-          <dt>void activateSender()</dt>
+          <dt>void setDirection(RTCRtpTransceiverDirection direction)</dt>
 
           <dd>
-            <p>The <dfn id="dom-rtptransceiver-activatesender"><code>activateSender</code></dfn>
-            method activates the <code>RTCRtpTransceiver.sender</code> such that future calls
-            to <code>createOffer</code> and <code>createAnswer</code> mark the corresponding
-            media description as <code>sendrecv</code> or <code>sendonly</code>, as defined in
-            <span data-jsep="subsequent-offers subsequent-answers">[[!JSEP]]</span>.
-
-            <p>If the sender is not already active and the PeerConnection is in the stable state,
-            this will set the <code>negotiationneeded</code> flag.
-          </dd>
-          
-          <dt>void activateReceiver()</dt>
-
-          <dd>
-            <p>The <dfn id="dom-rtptransceiver-activatereceiver"><code>activateReceiver</code></dfn>
-            method activates the <code>RTCRtpTransceiver.receiver</code> such that future calls
-            to <code>createOffer</code> and <code>createAnswer</code> mark the corresponding
-            media description as <code>sendrecv</code> or <code>recvonly</code>, as defined in
-            <span data-jsep="subsequent-offers subsequent-answers">[[!JSEP]]</span>.
-
-            <p>If the receiver is not already active and the PeerConnection is in the stable state,
-            this will set the <code>negotiationneeded</code> flag.
+            <p>
+            	The <dfn id="dom-rtptransceiver-setdirection"><code>setDirection</code></dfn>
+            	method sets the direction of the <code>RTCRtpTransceiver.sender</code> and
+            	<code>RTCRtpTransceiver.receiver</code> such that future calls
```

Comment:
I think it sets the direction of the transceiver, not the sender and receiver, which changes whether the sender or receiver are active.